### ISSUE TITLE
Problematic unscheduled session labels on interim session details view

### ISSUE THREAD

**Issue by jennifer-richards** [ietf-tools/datatracker#4671]
### Describe the issue

The `ietf.meeting.views.session_details` view for an interim shows both scheduled and unscheduled sessions. These are labeled by their session time or meeting time, depending on what is available. For scheduled interims, this is the local time if a country is specified or UTC time For unscheduled sessions, this is the meeting date. This selection is made in the view.

The tzaware branch displays the chosen time in meeting time zone using the `timezone` template filter. This creates at least a couple problems
* For unscheduled interims, where the "time" is actually a date object, the timezone template filter fails and the result is no label at all
* For scheduled interims, time is now always shown in meeting time because the tzinfo attached by the view is ignored

To resolve, need to decide what the desired behavior is and update the implementation to match that.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
While working out what the right thing to do is, remember to take edge cases like https://datatracker.ietf.org/meeting/interim-2020-teep-03/session/teep into account.

**Comment by rjsparks** [comment]
Generally:
* Don't show materials in unscheduled sessions when there are scheduled sessions and materials have the "apply to all session in meeting" configured. (This may require back-computation to see if the sessionpresentation sets are the same).
* If there are no materials to show, group the remaining unscheduled sessions by name and just list names with counts. This will be only be really useful for groups like `ietf` that have a bunch of unrelated sessions connected to them.

**Comment by rjsparks** [comment]
@jennifer-richards Is this still in need of change?

**Comment by jennifer-richards** [comment]
Yes, I think so. At least, it still blindly calls `time|timezone:session.meeting.time_zone`, even if `time` is a `datetime.date`. 

I'm not sure why I was concerned that "the tzinfo attached by the view is ignored." The timestamp for a scheduled session is shown in meeting time with the time zone labeled. Since there's no UI to change the display time zone, I think that's fine.

So probably just keep it from showing nothing when unscheduled?

**Comment by rjsparks** [comment]
Updating my comment based on preparation to address this and several other issues:

We should _show_ materials in unscheduled sessions if they exist. What we should not do is attach more to them if apply to all is selected - we should only apply to actually scheduled sessions. We shouldn't allow drag-drop of slides into the unscheduled sessions, but _should_ allow drag out of them.
